src/components/dialogs/input-hooks.js

```diff
@@ -412,25 +413,27 @@ export const useAutocompleteField = ({
     previousValue,
     loading = false,
 }) => {
+    const [presentedOptions, setPresentedOptions] = useState([]);
+    const [isLoading, setIsLoading] = useState(loading);
     const [value, setValue] = useState(defaultValue);
-    const [error, setError] = useState('');
+    const [error, setError] = useState();
     const validationRef = useRef();
     validationRef.current = validation;
 
     useEffect(() => {
-        if (defaultValue) {
-            setValue(defaultValue);
-        }
-    }, [defaultValue]);
+        setValue(defaultValue);
+    }, [defaultValue, values]);
 
     useEffect(() => {
         function validate() {
             const res = validateField(value, validationRef.current);
             setError(res?.errorMsgId);
             return !res.error;
         }
-        inputForm.addValidation(id ? id : label, validate);
-    }, [label, validation, inputForm, value, id]);
+        if (inputForm) {
+            inputForm.addValidation(id ? id : label, validate);
+        }
+    }, [label, validation, inputForm, value, id, error, errorMsg]);
```

Comment:
why error and errorMsg in the dependency array ?

Reply:
Removed